requirements.txt

```diff
@@ -1,5 +1,5 @@
 PyYAML==5.1
 argparse==1.1
 msgpack-python==0.4.6
-gevent==1.1b5
+gevent==21.1.1
```

Comment:
The change is okay, but the explanation 'gevent-1.1b5 is incompatible with Mac OS X 15' is wrong. It is not related to Mac OS. gevent 1.2 and below does not support Python 3.7 and above. See https://github.com/gevent/gevent/issues/1297

Reply:
Updated commit message.